examples/verify_and_create_letters_from_csv/input.csv

```diff
@@ -1,4 +1,4 @@
-name,amount,address1,address2,city,state,zip
+name,amount,primary_line,secondary_line,city,state,zip
```

Comment:
nit: We should change `zip` to `zip_code` since that's what we call it in the API